### PR TITLE
docs: add a vite sample to showcase components

### PR DESCRIPTION
**Pull request by JLou**
Added a sample to be able to quickly test out component in a "real-world" application and check exports.
It can also serve as a showcase of the components

### CODE REVIEW

**Comment by JLou** [comment]
waiting for #139 to be merge to change import of icons

File: package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
   "homepage": "https://github.com/AxaFrance/design-system#readme",
   "workspaces": [
     "packages/css",
-    "packages/react"
+    "packages/react",
+    "samples/vite"
```
Comment (pplancq) [inline package.json:25]: il n'y a pas un risque que la CI publish ce package ?

Comment (JLou) [reply package.json:25]: nope, j'ai également touché aux scripts pour que ça n'arrive pas

**Review by fffan64 — APPROVED** [review]

File: samples/vite/.eslintrc.cjs
Comment (pplancq) [inline samples/vite/.eslintrc.cjs:1]: On peux juste import la config ESLint global non ?

**Review by pplancq — APPROVED** [review]

**Review by johnmeunier — APPROVED** [review]

**Pull request merged**